src/Package.php

```diff
@@ -94,6 +138,23 @@ public function getMessages()
         return $this->messages;
     }
 
+
+    /**
+     *
+     * Gets the message of the given key for this package.
+     *
+     * @param string $key the key of the message to return
+     *
+     * @return string|null
+     *
+     */
+    public function getMessage($key)
+    {
+        if (isset($this->messages[$key])) {
+            return $this->messages[$key];
+        }
```

Comment:
PHP returns null by default, but we can make it more explicit, yes.

Actually, maybe if we want to make it more coherent with `Translator::getMessage();` we should return false instead ?